inc/classes/class_masterform.php

```diff
@@ -40,42 +40,32 @@ class masterform
     /**
      * @var array
      */
-    public $FormFields = [];
+    private $FormFields = [];
 
     /**
      * @var array
      */
-    public $Groups = [];
+    private $Groups = [];
 
     /**
      * @var array
      */
-    public $SQLFields = [];
+    private $SQLFields = [];
 
     /**
      * @var array
      */
-    public $WYSIWYGFields = [];
+    private $WYSIWYGFields = [];
 
     /**
      * @var array
      */
-    public $SQLFieldTypes = [];
```

Comment:
SQLFieldTypes is used within class_mastersearch2.php and thus declaration shouldn't be removed